docs/command/atlas-organizations-invitations-invite.txt

```diff
@@ -56,7 +56,7 @@ Options
    * - -f, --file
      - string
      - false
-     - Path to an optional JSON configuration file that defines invitation settings. To learn more about invitation configuration files for the Atlas CLI, see https://XXX.
+     - Path to an optional JSON configuration file that defines invitation settings. Note: Unsupported fields in the JSON file are ignored. To learn more about invitation configuration files for the Atlas CLI, see https://XXX.
```

Comment:
👀 
>see https://XXX.

Reply:
Good catch, I've removed the URL for now, since there's no documentation yet.
I also created a DOCSP ticket to create this page: https://jira.mongodb.org/browse/DOCSP-47853